content/telegraf/v1.19/about_the_project/release-notes-changelog.md

```diff
@@ -10,14 +10,42 @@ menu:
     parent: About the project
 ---
 
+## v.1.19.3 [2021-08-19]
+
+- Update `sirupsen/logrus` module from 1.7.0 to 1.8.1.
+- Update `testcontainers/testcontainers-go` module from 0.11.0 to 0.11.1.
+- Update `golang/snappy` module from 0.0.3 to 0.0.4.
+- Update `aws/aws-sdk-go-v2` module from 1.3.2 to 1.8.0.
+- Update `sensu/go` module to v2.9.0.
+- Update `hashicorp/consul/api` module to 1.9.1.
+
+### Input plugin updates
+- Prometheus (`prometheus`): Fix Kubernetes pod discovery.
+- Redis (`redis`) Improve redis commands documentation.
+- Clickhouse (`clickhouse`): Fix panic, improve handling empty result set.
+- OPC UA: (`opcua`):
+  - Avoid closing session on a closed connection.
+  - Fix reconnection regression introduced in 1.19.1.
+  - Don't skip good quality nodes after encountering bad quality node.
+- Kubernetes Inventory (`kube_inventory`): Fix k8s nodes and pods parsing error.
+- Postgre SQL (`postgresql`): Normalize unix socket path.
+- vSphere (`vsphere`): Update `vmware/govmomi` module to v0.26.0 in order to support vSphere 7.0.
+
+### Output plugin updates
+- Loki (`loki`): Sort logs by timestamp before writing to Loki.
+- CrateDB (`cratedb`): Replace dots in tag keys with underscores.
+
+### Processor plugin updates
+- AWS (`aws`): Refactor EC2 init for `config-api`.
```

Comment:
maybe remove "for config-api". this is a sorta new feature thing that not many people know about. so "Refactor EC2 init" should be fine